packages/signing-utils/src/utils.ts

```diff
@@ -0,0 +1,16 @@
+import { debug as debugFn } from 'debug';
+
+export const debug = debugFn('signing-utils');
+
+export function assertRequiredVars() {
```

Comment:
I think this might no longer be relevant-I changed the file to gather environment variables needed for code signing. It is now used in multiple locations.